pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -444,5 +435,27 @@ stdenv.mkDerivation ((drvAttrs config stdenv.hostPlatform.linux-kernel kernelPat
   ] ++ (stdenv.hostPlatform.linux-kernel.makeFlags or [])
     ++ extraMakeFlags;
 
-  karch = stdenv.hostPlatform.linuxArch;
-} // (optionalAttrs (pos != null) { inherit pos; })))
+  finalKernel = stdenv.mkDerivation (
```

Comment:
lib.fix?

Reply:
We could, I was kinda worried about making things slower though.

Reply:
Oh, I'm surprised it would make a difference. Okay.